modules/n1ql/examples/transactions/single.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# tag::query[]
+curl http://localhost:8093/query/service \
+-u Administrator:password \
```

Comment:
indentation missing after line continuation? (may be one for our styleguide meeting)